src/ui/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,62 @@
+import * as AvatarPrimitive from '@radix-ui/react-avatar';
+
+import { styled } from '../../stitches.config';
+import { getAvatarPathWithFallback } from 'utils/domain';
+
+import { IUser, IProfile } from 'types';
+
+const AvatarRoot = styled(AvatarPrimitive.Root, {
+  display: 'inline-flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  verticalAlign: 'middle',
+  overflow: 'hidden',
+  userSelect: 'none',
+  width: 45,
+  height: 45,
+  borderRadius: '100%',
+  backgroundColor: 'Black',
+});
+
+const AvatarImage = styled(AvatarPrimitive.Image, {
+  width: '100%',
+  height: '100%',
+  objectFit: 'cover',
+  borderRadius: 'inherit',
+});
+
+const AvatarFallback = styled(AvatarPrimitive.Fallback, {
+  width: '100%',
+  height: '100%',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  backgroundColor: 'white',
+  color: 'Violet',
+  fontSize: 15,
+  lineHeight: 1,
+  fontWeight: 500,
+});
+
+export const Avatar = ({
+  user,
+  path,
+  profile,
+}: {
+  user?: IUser;
+  path?: string;
+  profile?: IProfile;
+}) => {
+  const avatarPath = getAvatarPathWithFallback(
+    profile?.avatar || user?.profile?.avatar,
+    user?.name
+  );
+  const src = path ?? avatarPath;
+
+  return (
+    <AvatarRoot>
+      <AvatarImage src={src} alt={user?.name} />
+      <AvatarFallback>avatarPath</AvatarFallback>
```

Comment:
?

Reply:
If you asking about the component, this represents the fallback to failure situation to display just the initials.

If you asking about using `avatarPath` as a fallback, the `getAvatarPathWithFallback()` will return initials in case of path absence. 

https://github.com/coordinape/coordinape/blob/258fad5b4572ed35324f562ec75e086c7f483f70/src/utils/domain.ts#L56-L60

Besides, I'm not fan of this implementation; It knows too much about the logic behind determining path and user. I would change it to more generic function that accepts only path and initials and change `ApeAvatar` to utilize it instead. What do you think?

Reply:
@levity I updated `ui/Avatar` to only accept `path` & `name`.

Reply:
Thanks for updating the avatar. My "?" comment was about the presence of the string literal "avatarPath" here. I think you're just missing curly braces? But even then I don't understand why you would print the path. Maybe I'm missing something about how AvatarFallback works.

Reply:
Yes, my mistake. It's fixed now.
In addition, `getAvatarPathWithFallback()` is kinda confusing. I should think of better naming.